.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,68 @@
+---
+name: 🐛 Bug Report
+description: Report a reproducible bug in the current release of pynetbox
+labels: ["type: bug"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        **NOTE:** This form is only for reporting _reproducible bugs_ in a current pynetbox release.
+  - type: input
+    attributes:
+      label: pynetbox version
+      description: What version of pynetbox are you currently running?
+      placeholder: v7.1.0
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: NetBox version
+      description: What version of NetBox are you currently running?
+      placeholder: v3.5.8
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Python version
+      description: What version of Python are you currently running?
+      options:
+        - "3.8"
+        - "3.9"
+        - "3.10"
+        - "3.11"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to Reproduce
+      description: >
+        Please provide a minimal working example to demonstrate the bug. Begin with the 
+        initialization of any necessary database objects and clearly enumerate each 
+        operation carried out. Ensure that your example is as concise as possible 
+        while adequately illustrating the issue. For example:
+        ```python
+        >>> import pynetbox
+        >>> nb = pynebox.api('https://netbox.example.com', token='my-token')
+        
+        ```
+        Note: **do not utilize the demo instance** for replicating suspected bugs, 
+        as its data is subject to change or removal at any time.
+        
+        _Please refrain from including any confidential or sensitive 
+        information in your example._
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: What did you expect to happen?
+      placeholder: A new widget should have been created with the specified attributes
```

Comment:
Maybe update this comment, widgets don't make sense in pynetbox

Reply:
Check now, does that look better?